src/Midgard/CreatePHP/Mapper/BaseDoctrineRdfMapper.php

```diff
@@ -58,9 +61,10 @@ public function store(EntityInterface $entity)
      *
      * use getRealClass if className names a doctrine proxy class.
      */
-    public function canonicalName($className)
```

Comment:
for BC, lets just keep this method as it was but add the new objectToName method.

we can then eventually go for a version 2.0 of this library, but for now lets keep things working.